### PR TITLE
Add option to disable preferred batch size sweep

### PR DESCRIPTION
**Pull request by kthui**

### CODE REVIEW

**Comment by kthui** [comment]
There are many styling script updates to `model_analyzer/config/input/config_command_profile.py` file. The real changes are `Line 38, 435 - 441`.

**Review by Tabrizian — CHANGES_REQUESTED** [review]
Can you add a test for this in `L0_config_search` as well?

**Comment by kthui** [comment]
Yes. Will work on it.

File: qa/L0_config_search/test_config_generator.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        self._write_file(2, 1, 2, 1, model_config)
+
+    def generate_max_limit_with_dynamic_batch_disable(self):
+        model_config = {
+            "run_config_search_max_concurrency": 2,
+            "run_config_search_max_instance_count": 2,
+            "run_config_search_preferred_batch_size_disable": True,
+            "profile_models": self.profile_models,
+        }
+        self._write_file(4, 2, 4, 1, model_config)
```
Comment (kthui) [inline qa/L0_config_search/test_config_generator.py:139]: New test case for disabling preferred batch size sweep.

**Review by Tabrizian — APPROVED** [review]

**Comment by kthui** [comment]
Merged, as the next PR is likely to step on this one.

**Pull request merged**